doc/source/_toc.yml

```diff
@@ -296,25 +292,13 @@ parts:
           - file: cluster/vms/examples/ml-example
         - file: cluster/vms/references/index
       - file: cluster/running-applications/index
-        title: Running Applications on Ray Clusters
-      - file: cluster/references/index
-        title: References
+        title: Applications Guide
 
-      - file: cluster-deprecated/index
-        title: Deprecated Ray Clusters Docs
-        sections:
-        - file: cluster-deprecated/key-concepts
-        - file: cluster-deprecated/cloud
-        - file: cluster-deprecated/quickstart
-        - file: cluster-deprecated/usage-stats
-        - file: cluster-deprecated/user-guide
-        - file: cluster-deprecated/cluster_under_construction.md
-        - file: cluster-deprecated/deploy
-        - file: cluster-deprecated/api
 
   - caption: References
     chapters:
       - file: ray-references/api
+      - file: usage-stats
```

Comment:
Warning: [hardcoded references](https://github.com/ray-project/ray/blob/master/python/ray/_private/usage/usage_constants.py#L21) to `https://docs.ray.io/en/master/cluster/usage-stats.html` exist.

Reply:
Thanks for the reminder; moved it back.